Sources/joyfill-components-swift/Components/Table/ViewTable.swift

```diff
@@ -332,6 +360,17 @@ public class ViewTable: UIViewController, TextFieldCellDelegate, TextViewCellDel
         
         let tap = UITapGestureRecognizer(target: self, action: #selector(mainViewTap))
         mainView.addGestureRecognizer(tap)
+        
+        // Sets table and action to pageButton
+        if #available(iOS 13.0, *) {
```

Comment:
this comment _"// Sets table and action to pageButton"_ doesn't tell me why we have to have an if change for iOS 13.0. Why do we have this if and can we make it more clear why this is needed?

Reply:
don't think you dealt with this one, please address all comments

Reply:
I dealt with this one as well.
I removed that comment because it is not required there and this version check is required to give systemImages.